src/web/crate_details.rs

```diff
@@ -122,6 +124,8 @@ impl CrateDetails {
         req_version: Option<ReqVersion>,
         prefetched_releases: Vec<Release>,
     ) -> Result<Option<CrateDetails>, anyhow::Error> {
+        // FIXME: Since all subqueries from `builds` table have the same conditions, would be
```

Comment:
We could do it right now? 

Here is an example: 
https://github.com/rust-lang/docs.rs/blob/05b70cf438e3ac091dcc6f5acbff0a53f1cc2c9d/src/web/mod.rs#L686-L691

Reply:
Damn, it was the `LATERAL` part that I was missing. Thanks for the help!